### PR TITLE
Add support for CASP 1.0

### PR DESCRIPTION
**Pull request by imobachgs**

### CODE REVIEW

File: lib/aytests/iso_repo.rb [hunk 1]
```diff
@@ -35,28 +35,18 @@ def initialize(dir)
...
-      if download_to(uri)
-        # Returning the first found ISO in this directory
-        filename = Dir.entries(iso_dir).find do |f| 
-          File.file?(File.join(iso_dir, f)) && f.end_with?(".iso")
-        end
-        File.join( iso_dir, filename)
-      else
-        false
-      end
+      return false unless download_to(uri)
```
Comment (imobachgs) [inline lib/aytests/iso_repo.rb]: Yes, the main point of this change is that previously it deleted the ISOs before downloading the new one. And, from Canary Islands, it was painful :)

**Comment by schubi2** [comment]
still LGTM :-)

**Comment by schubi2** [comment]
still LGTM :-)

**Comment by imobachgs** [comment]
Thanks again :)

**Pull request merged**